.github/workflows/build_packages.yaml

```diff
@@ -293,12 +317,16 @@ jobs:
     - name: load rocksdb cache
       uses: actions/cache@v2
       with:
-        path: source/_build/default/lib/rocksdb/
+        path: |
+          source/_build/default/lib/rocksdb/
+          source/deps/rocksdb/
```

Comment:
Q: is this the right syntax ? 

if multiple paths are supported, it's more likely to be: 
```
        path:
           - source/_build/default/lib/rocksdb/
           - source/deps/rocksdb/
```

Reply:
never mind, googled it.

Reply:
Yes, I took it from here: https://github.com/marketplace/actions/cache#whats-new

At least it seems to be working. :smile_cat: